blue-note/theme.json

```diff
@@ -464,9 +519,9 @@
 		"spacing": {
 			"padding": {
 				"top": "0",
-				"right": "var(--wp--preset--spacing--60)",
+				"right": "var(--wp--preset--spacing--50)",
```

Comment:
I'd suggest going with 40 left and right here.

Reply:
The design uses a minimum of 61px on desktop.

Reply:
Spacing size 40, not pixels. And I’d personally tune it to 60px, instead of 61.

Reply:
The current mobile width is a bit too wide as-is.

Reply:
Yes,  I understood that you meant the spacing preset =D

Spacing size 40 is a minimum of 40px until 6vw is smaller, no?

```

				{
					"size": "min(40px, 6vw)",
					"slug": "40",
					"name": "4"
				},

				{
					"size": "min(60px, 12vw)",
					"slug": "60",
					"name": "6"
				},
```

The content is not centered on the mobile designs, and some are 11 px on the left and 20 on the right. So I assume we have some freedom to make a decision and not follow the exact design.
I don't get a chance to work with a design that someone else has created very often, so I am always weary about how much I can change.